components/templates/templates.js

```diff
@@ -64,7 +66,7 @@ if (!utils.templates) {
     */
     function getTemplate(target, data) {
       var template;
-      var templates = target.querySelectorAll('*[data-template]');
+      var templates = target.querySelectorAll('template');
```

Comment:
I realized that we have to change this in two places, maybe a constant to share the selector could be useful but only if you consider this interesting